draft-ietf-ccwg-bbr.md

```diff
@@ -1096,15 +1089,7 @@ new data, and when the delivery rate sample is not application-limited
       BBR.full_bw_reached = true
 ~~~~
 
-BBR waits three rounds to have solid evidence that the sender is not detecting
-a delivery-rate plateau that was temporarily imposed by the receive window.
-Allowing three rounds provides time for the receiver's receive-window auto-tuning
-to open up the receive window and for the BBR sender to realize that BBR.max_bw
-should be higher: in the first round the receive-window auto-tuning algorithm
-grows the receive window; in the second round the sender fills the higher
-receive window; in the third round the sender gets higher delivery-rate samples.
-This three-round threshold was validated by YouTube experimental data.
-
+BBR waits three packet-timed round trips to have reasonable evidence that the sender is not detecting a delivery-rate plateau that was temporarily imposed by congestion or receive-window auto-tuning. This three-round threshold was validated by experimental data to allow the receiver the chance to grow its receive window.
```

Comment:
The text LG, but usually we reflow markdown to ~80 character lines, at least in my experience.

Reply:
I prefer 80 character lines as well. It should make it easier to have smaller, easy-to-review diffs. And make it easier to view/edit as well.

@jbeshay - if you have time to reformat to 80 characters, that's great.

Since we're on a deadline, I'm OK with merging this version now and I can reformat as a follow-up if you want.

Reply:
Sounds good. I'll merge this now then. I will push the reformat to 80 characters later tonight.